bin/web/views/index.ejs

```diff
@@ -319,19 +358,19 @@ ${fields}
         //     }
         // })
         document.querySelector("#prompt-cancel").addEventListener("click", (e) => {
-          e.preventDefault()
-          e.stopPropagation()
-          socket.emit("request", {
-            prompt: "/stop"
-          })
-          document.querySelector("#prompt-run").classList.remove("hidden")
-          document.querySelector("#prompt-cancel").classList.add("hidden")
+            e.preventDefault()
+            e.stopPropagation()
+            socket.emit("request", {
+                prompt: "/stop"
+            })
+            document.querySelector("#prompt-run").classList.remove("hidden")
+            document.querySelector("#prompt-cancel").classList.add("hidden")
         })
 
         // Load prompts from files
 
         // Update the input text with the selected prompt value
-        const handlePromptChange = () => {
+        const updateInputToPrompt = () => {
```

Comment:
i renamed this because we are calling it whenever we want to "reset" the input into the prompt.. including after submitting your input